pubspec.yaml

```diff
@@ -1,10 +1,10 @@
 name: form_validator
 description: Simplest form validation library for flutter's form field widgets
-version: 0.1.7
+version: 0.2.0
```

Comment:
We have to increase major version since the package will not be compatible with previous versions of flutter. So can you please set version to "1.0.0" ?

Reply:
@TheMisir I chose 0.2.0 by following [their migration documentation](https://dart.dev/null-safety/migration-guide#package-version), which says "If your package hasn’t reached 1.0.0 yet, either increase the minor version or update the version to 1.0.0. For example, if the previous version is 0.3.2, the new version is either 0.4.0 or 1.0.0.".  Happy to change it to 1.0.0 if you feel this is a 1.0.0 release though - just let me know!

Reply:
this is correct